pkg/session_recording/recordings.jsx

```diff
@@ -304,8 +304,8 @@
             return (
                 <div>
                     <div className="content-header-extra">
-                        <table class="form-table-ct">
-                            <tr>
+                        <table className="form-table-ct">
```

Comment:
Unfortunately, the commit subject is still confusing. Did you mean something like "Use className attr in recording list table"?

Sorry, leaving a comment for commit message in the code. GitHub doesn't support commenting on commit message yet, and I'm in a hurry.